Changes

```diff
@@ -23,6 +23,9 @@ Language features:
 - GPR#88: allow field punning in object copying expressions:
     {< x; y; >} is sugar for {< x = x; y = y; >}
   (Jeremy Yallop)
+* GPR#234: allow "[]" as a user-defined type constructor. Demand parenthesis
+  around "::" when using "::" as type constructor.
```

Comment:
"type constructor" is the wrong phrase here. You should just say "user-defined constructor" (in both instances).